XiEditor/Trace.swift

```diff
@@ -17,6 +17,10 @@
 
 import Foundation
 
+protocol Writeable {
```

Comment:
it was at one point before I added snapshot().  Now the entire write() method is superfluous since we send the samples via RPC which auto-magically JSONifies them.  Removed this + the write function.